packages/react/src/themes/teams/components/Dialog/dialogStyles.ts

```diff
@@ -16,6 +16,13 @@ export default {
     gridTemplateColumns: '1fr auto',
     boxShadow: v.boxShadow,
     color: v.foregroundColor,
+
+    [v.rootWidthBreakpointMedium]: {
```

Comment:
Shouldn't we check whether it is defined first?

Reply:
It's required by typings. Why we need to have additional checks there?